### PR TITLE
Create a container that uses Redux to display a list oof markdown files

### PR DESCRIPTION
**Pull request by jwmcconnell**

### CODE REVIEW

File: coverage/clover.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mehtaphysical) [inline coverage/clover.xml]: whats this for? can we .gitignore it?

File: coverage/coverage-final.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{"/Users/jackadmin/Documents/github/markdownly/src/actions/documentActions.js": {"path":"/Users/jackadmin/Documents/github/markdownly/src/actions/documentActions.js","statementMap":{"0":{"start":{"line":1,"column":31},"end":{"line":1,"column":48}},"1":{"start":{"line":2,"column":30},"end":{"line":5,"column":2}},"2":{"start":{"line":2,"column":45},"end":{"line":5,"column":1}},"3":{"start":{"line":7,"column":28},"end":{"line":7,"column":42}},"4":{"start":{"line":8,"column":27},"end":{"line":11,"column":2}},"5":{"start":{"line":8,"column":39},"end":{"line":11,"column":1}}},"fnMap":{"0":{"name":"(anonymous_0)","decl":{"start":{"line":2,"column":30},"end":{"line":2,"column":31}},"loc":{"start":{"line":2,"column":45},"end":{"line":5,"column":1}},"line":2},"1":{"name":"(anonymous_1)","decl":{"start":{"line":8,"column":27},"end":{"line":8,"column":28}},"loc":{"start":{"line":8,"column":39},"end":{"line":11,"column":1}},"line":8}},"branchMap":{},"s":{"0":2,"1":2,"2":2,"3":2,"4":2,"5":2},"f":{"0":2,"1":2},"b":{},"_coverageSchema":"43e27e138ebf9cfc5966b082cf9a028302ed4184","hash":"14e2346123612cccd28f913543e5a4c315c378c3"}
```
Comment (mehtaphysical) [inline coverage/coverage-final.json]: .gitignore

File: coverage/lcov-report/actions/documentActions.js.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
+<!doctype html>
```
Comment (mehtaphysical) [inline coverage/lcov-report/actions/documentActions.js.html]: .gitignore

File: coverage/lcov-report/actions/index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
+<!doctype html>
```
Comment (mehtaphysical) [inline coverage/lcov-report/actions/index.html]: .gitignore

File: coverage/lcov-report/base.css [hunk 5]
```diff
@@ -0,0 +1,224 @@
+body, html {
```
Comment (mehtaphysical) [inline coverage/lcov-report/base.css]: .gitignore

**Comment by mehtaphysical** [comment]
looks good. gitignore the coverage stuff.

**Comment by Zilula** [comment]
LGTM!

**Review by Zilula — APPROVED** [review]

**Review by dannyhogan — APPROVED** [review]
Github savants

**Pull request merged**